frappe/desk/doctype/dashboard_chart/test_dashboard_chart.py

```diff
@@ -40,21 +40,22 @@ def test_dashboard_chart(self):
 		if frappe.db.exists('Dashboard Chart', 'Test Dashboard Chart'):
```

Comment:
Make two tests, one for doctype and one for JSON. Use dashboard chart source too